src/templates/entity/entity.template.ts

```diff
@@ -0,0 +1,124 @@
+import { allOfDereference } from '@templates/lib/allOfdereference'
+import { generateTsFile } from '@utils/generateTsFile'
+import { getFileImports } from '@utils/getFileImports'
+import * as _ from 'lodash'
+
+class EntityFileFactory {
+    private enums: Record<string, any> = {}
+
+    private imports: Record<string, Set<string>> = {
+        ['typeorm']: new Set(),
+        ['@nestjs/swagger']: new Set(),
+    }
+
+    private rootPath: string
+
+    private serviceName: string
+
+    private entities: { title: string; code: string }[]
+
+    constructor(rootPath: string, serviceName: string, entitySchemas: [string, any][]) {
+        this.rootPath = rootPath
+        this.serviceName = serviceName
```

Comment:
```suggestion
    constructor(private rootPath: string, private serviceName: string, entitySchemas: [string, any][]) {
```

можно использовать более короткую запись инициализации

Reply:
left as is. по-моему исходная версия более наглядная без каких-то заморочек... с точки зрения будущего сопровождения кода - чем менее мудреный синтаксис - тем проще...